x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/support/CachingUsernamePasswordRealm.java

```diff
@@ -148,11 +148,8 @@ private void doAuthenticateAndCache(UsernamePasswordToken token, ActionListener<
         doAuthenticate(token, wrapped);
     }
 
-    @Override
-    public Map<String, Object> usageStats() {
-        Map<String, Object> stats = super.usageStats();
-        stats.put("size", cache.count());
```

Comment:
I feel like we should keep this stat as `cache` (or some other key), rather than removing it entirely.
(a) it allows for a comparison between old & new
(b) it is indicative of _usage_ in the sense of how many users have recently logged in.